ped/bashcomplete.py

```diff
@@ -0,0 +1,14 @@
+from __future__ import print_function
+
+import os
+import shutil
+
+def install():
+    # Is this right for all systems?
+    destination = '/etc/bash_completion.d/'
```

Comment:
Perhaps instead of trying to guess where the file should go, we require the user to specify the path, i.e.

```
python -m ped.install_completion /etc/bash_completion.d/
```